<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="Onboarding_test_tenant-20231219121541" tests="1" file=".py" time="196.422" timestamp="2023-12-19T12:18:57" failures="0" errors="1" skipped="0">
	<testcase classname="Onboarding_test_tenant" name="test_tenat_onboarding" time="196.422" timestamp="2023-12-19T12:18:57" file="Onboarding.py" line="36">
		<error type="NoSuchWindowException" message="Message: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=120.0.6099.110)
Stacktrace:
	GetHandleVerifier [0x00007FF794452142+3514994]
	(No symbol) [0x00007FF794070CE2]
	(No symbol) [0x00007FF793F176AA]
	(No symbol) [0x00007FF793EF0AFD]
	(No symbol) [0x00007FF793F8CB1B]
	(No symbol) [0x00007FF793FA218F]
	(No symbol) [0x00007FF793F85D93]
	(No symbol) [0x00007FF793F54BDC]
	(No symbol) [0x00007FF793F55C64]
	GetHandleVerifier [0x00007FF79447E16B+3695259]
	GetHandleVerifier [0x00007FF7944D6737+4057191]
	GetHandleVerifier [0x00007FF7944CE4E3+4023827]
	GetHandleVerifier [0x00007FF7941A04F9+689705]
	(No symbol) [0x00007FF79407C048]
	(No symbol) [0x00007FF794078044]
	(No symbol) [0x00007FF7940781C9]
	(No symbol) [0x00007FF7940688C4]
	BaseThreadInitThunk [0x00007FFBA8B3257D+29]
	RtlUserThreadStart [0x00007FFBA8E8AA58+40]
"><![CDATA[Traceback (most recent call last):
  File "c:\Users\luist\repos\Automatizaciones-SILOHUB-\Onboarding.py", line 106, in test_tenat_onboarding
    cuit = CUITGenerator.generate_random_cuit(self.driver, input_xpath)
  File "c:\Users\luist\repos\Automatizaciones-SILOHUB-\GeneradorCuit.py", line 19, in generate_random_cuit
    input_element = driver.find_element(By.XPATH, input_xpath)
  File "C:\Users\luist\AppData\Roaming\Python\Python310\site-packages\selenium\webdriver\remote\webdriver.py", line 741, in find_element
    return self.execute(Command.FIND_ELEMENT, {"using": by, "value": value})["value"]
  File "C:\Users\luist\AppData\Roaming\Python\Python310\site-packages\selenium\webdriver\remote\webdriver.py", line 347, in execute
    self.error_handler.check_response(response)
  File "C:\Users\luist\AppData\Roaming\Python\Python310\site-packages\selenium\webdriver\remote\errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchWindowException: Message: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=120.0.6099.110)
Stacktrace:
	GetHandleVerifier [0x00007FF794452142+3514994]
	(No symbol) [0x00007FF794070CE2]
	(No symbol) [0x00007FF793F176AA]
	(No symbol) [0x00007FF793EF0AFD]
	(No symbol) [0x00007FF793F8CB1B]
	(No symbol) [0x00007FF793FA218F]
	(No symbol) [0x00007FF793F85D93]
	(No symbol) [0x00007FF793F54BDC]
	(No symbol) [0x00007FF793F55C64]
	GetHandleVerifier [0x00007FF79447E16B+3695259]
	GetHandleVerifier [0x00007FF7944D6737+4057191]
	GetHandleVerifier [0x00007FF7944CE4E3+4023827]
	GetHandleVerifier [0x00007FF7941A04F9+689705]
	(No symbol) [0x00007FF79407C048]
	(No symbol) [0x00007FF794078044]
	(No symbol) [0x00007FF7940781C9]
	(No symbol) [0x00007FF7940688C4]
	BaseThreadInitThunk [0x00007FFBA8B3257D+29]
	RtlUserThreadStart [0x00007FFBA8E8AA58+40]

]]></error>
		<system-out><![CDATA[El texto obtenido es: DOCUMENTACIÓN
El texto obtenido es: Para poder continuar al siguiente paso, será obligatorio completar los campos que se muestran a continuación.
]]></system-out>
	</testcase>
</testsuite>
